async function main() {}

main()
  .then(() => console.log('done'))
  .catch((e) => console.error(e));
